ts/components/conversation/media-gallery/MediaGallery.tsx

```diff
@@ -0,0 +1,148 @@
+/**
+ * @prettier
+ */
+import React from 'react';
+
+import moment from 'moment';
+
+import { AttachmentSection } from './AttachmentSection';
+import { groupMessagesByDate } from './groupMessagesByDate';
+import { ItemClickEvent } from './events/ItemClickEvent';
+import { Message } from './propTypes/Message';
+
+type AttachmentType = 'media' | 'documents';
+
+interface Props {
+  documents: Array<Message>;
+  i18n: (key: string, values?: Array<string>) => string;
+  media: Array<Message>;
+  onItemClick?: (event: ItemClickEvent) => void;
+}
+
+interface State {
+  selectedTab: AttachmentType;
+}
+
+const MONTH_FORMAT = 'MMMM YYYY';
+const COLOR_GREY = '#f3f3f3';
```

Comment:
Will be so nice to get crisp on all of our greys! And maybe start spelling it as nature intended, 'gray'! :0)

Reply:
LOL, this one always gets me! 🇺🇸😆